contract_isp/__init__.py

```diff
@@ -3,8 +3,9 @@
 ##############################################################################
 #
```

Comment:
Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3